versioned_docs/version-v1.4/tutorials/helm-multi-cluster.md

```diff
@@ -317,6 +317,82 @@ If you're using the `velaux` UI console, you can get even more information with
 
 ![resource-detail](../resources/helm-topology.jpg)
 
+## Specify different value file for different environment
+
+You can choose different value file present in a helm chart for different environment. eg:
+
+First guarantee your local cluster have two namespaces "test" and "prod" which represent test and prod environment.
+
+Then deploy app:
+
+```yaml
+cat <<EOF | vela up -f -
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: hello-kubernetes
+spec:
+  components:
+    - name: hello-kubernetes
+      type: helm
+      properties:
+        repoType: "helm"
+        url: "https://wangyikewxgm.github.io/my-charts/"
+        chart: "hello-kubernetes-chart"
+        version: "0.1.0"
+
+  policies:
+    - name: topology-test
+      type: topology
+      properties:
+        clusters: ["local"]
+        namespace: "test"
+    - name: topology-prod
+      type: topology
+      properties:
+        clusters: ["local"]
+        namespace: "prod"
+    - name: override-prod
+      type: override
+      properties:
+        components:
+          - name: hello-kubernetes
+            properties:
+              valuesFiles:
+                - "values-production.yaml"
```

Comment:
This use case needs to be explained:

```yaml
- name: override-test
      type: override
      properties:
        components:
          - name: hello-kubernetes
            properties:
              valuesFiles:
                - "values.yaml"
                - "values-test.yaml"
- name: override-prod
      type: override
      properties:
        components:
          - name: hello-kubernetes
            properties:
              valuesFiles:
                - "values.yaml"
                - "values-production.yaml"
```

Reply:
This doc has explained.

Reply:
Yes, we should explain what will happen if we write more than 1 files.